### ISSUE TITLE
Dependency resolution issue with the npm packages

### ISSUE THREAD

**Issue by FridIsar** [Abdulrahman-salem/React-app-Portfolio-MigraCode#28]
When doing:
`npm install`
I am getting a dependency resolution issue.
```
npm ERR! code ERESOLVE
23:23:38.004	npm ERR! ERESOLVE could not resolve
23:23:38.004	npm ERR! 
23:23:38.005	npm ERR! While resolving: react-scripts@5.0.1
23:23:38.005	npm ERR! Found: typescript@5.1.6
23:23:38.005	npm ERR! node_modules/typescript
23:23:38.005	npm ERR!   dev typescript@"^5.1.6" from the root project
23:23:38.006	npm ERR!   peer typescript@">= 2.7" from fork-ts-checker-webpack-plugin@6.5.3
23:23:38.006	npm ERR!   node_modules/fork-ts-checker-webpack-plugin
23:23:38.006	npm ERR!     fork-ts-checker-webpack-plugin@"^6.5.0" from react-dev-utils@12.0.1
23:23:38.006	npm ERR!     node_modules/react-dev-utils
23:23:38.007	npm ERR!       react-dev-utils@"^12.0.1" from react-scripts@5.0.1
23:23:38.007	npm ERR!       node_modules/react-scripts
23:23:38.007	npm ERR!         react-scripts@"5.0.1" from the root project
```
Temporary fix:
`npm install --force`
The above is bad practice and will prevent you from deploying the app correctly. 
Make sure the dependencies are fixed and that the app works with recent npm versions.

**Comment by Abdulrahman-salem** [comment]
Solved in New packages version #33 

Try now and If you find another problem, we will be pleased to solve it.

**Comment by FridIsar** [comment]
It now works! Thanks, I'll let you know
